src/docs/forms/components/CalendarInput/examples/04-iconClass.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import Form, {CalendarInput,RadioGroup} from "doj-react-adminlte/Form";
+
+export default class IconClass extends React.Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            selectedDate: null,
+            icons: "fa fa-calendar"
+        };
+    }
+
+    handleDateChange = (name, value) => {
+        this.setState({[name]: value});
+    };
+
+    render() {
+        const {selectedDate,icons} = this.state;
+        const iconOptions = [
+            {label: "Calendar", value: "fa fa-calendar"},
```

Comment:
Make the labels same as the value. (i.e., `label: "fa fa-calendar", value: "fa fa-calendar")`